src/jit/gentree.h

```diff
@@ -5131,86 +5131,146 @@ struct GenTreeILOffset : public GenTree
 struct Statement
 {
 public:
-    GenTree*       gtStmtExpr;      // root of the expression tree
-    GenTree*       gtStmtList;      // first node (for forward walks)
-    InlineContext* gtInlineContext; // The inline context for this statement.
-    IL_OFFSETX     gtStmtILoffsx;   // instr offset (if available)
-
+    Statement(GenTree* expr, IL_OFFSETX offset DEBUGARG(unsigned stmtID))
+        : m_rootTree(expr)
+        , m_treeList(nullptr)
+        , m_inlineContext(nullptr)
+        , m_ILOffsetX(offset)
 #ifdef DEBUG
-    IL_OFFSET gtStmtLastILoffs; // instr offset at end of stmt
-
-private:
-    unsigned m_stmtID;
+        , m_lastILOffset(BAD_IL_OFFSET)
+        , m_stmtID(stmtID)
 #endif
+        , m_next(nullptr)
+        , m_prev(nullptr)
+        , m_compilerAdded(false)
+    {
+    }
 
-public:
-    __declspec(property(get = getPrevStmt)) Statement* gtPrevStmt;
+    GenTree* GetRootTree() const
```

Comment:
"root tree" sounds a bit strange, shouldn't it be "tree root"/"root node"/"tree" instead?

Reply:
Thanks, it is hard for me to catch such moments.
I am not sure about the best name here,
"tree root"  - sounds like `Statement` is a tree class (like `class tree->GetRoot()`);
"root node" - this one I would prefer, but let's see what the others think.

for me "rootTree" shows that `Statement` contains tree nodes and one of them is a root.